test/e2e/framework/util.go

```diff
@@ -3900,9 +3900,7 @@ func sshRestartMaster() error {
 	}
 	var command string
 	if ProviderIs("gce") {
-		// `kube-apiserver_kube-apiserver` matches the name of the apiserver
-		// container.
-		command = "sudo docker ps | grep kube-apiserver_kube-apiserver | cut -d ' ' -f 1 | xargs sudo docker kill"
+		command = "sudo ps -ef | grep \"[k]ube-apiserver\" | awk '{print $2}' | xargs sudo kill"
```

Comment:
Why not just use `pidof`?

Reply:
Also, it might be nice to factor this out as a function. Not required by this PR though.

Reply:
Will do.